apps/web/app/future/video/[uid]/page.tsx

```diff
@@ -34,14 +35,10 @@ async function getData(context: GetServerSidePropsContext) {
       user: {
         select: {
           id: true,
+          username: true,
           timeZone: true,
           name: true,
           email: true,
-          organization: {
```

Comment:
We are going to remove organizationId and organization relation from User model. It's a preparation for that.

We would just be setting the organization relation on User but not reading it for any decision making. 

Setting it is necessary in case we need to revert the PR